File: src/Dr.NutrizioNino.WebVue/src/Interfaces/userProfile/SciaudoneCardDto.ts
/** Scheda Sciaudone calcolata da una misurazione: stima generica, non una prescrizione nutrizionale. */
export interface SciaudoneCardDto {
  id: string
  profileEntryId: string
  computedAt: string
  weightKg: number
  idealWeightKg: number
  kcal: number
  proteinG: number
  fatG: number
  fiberG: number
  carbsG: number
}

File: Dr.NutrizioNino.Models/Dto/Auth/SciaudoneCardDto.cs
namespace Dr.NutrizioNino.Models.Dto.Auth;

/// <summary>Scheda Sciaudone calcolata da una misurazione di profilo. Stima generica, non una prescrizione.</summary>
public record SciaudoneCardDto(
    Guid Id,
    Guid ProfileEntryId,
    DateTime ComputedAt,
    decimal WeightKg,
    decimal IdealWeightKg,
    decimal Kcal,
    decimal ProteinG,
    decimal FatG,
    decimal FiberG,
    decimal CarbsG
);
